Paws4Me/Paws4Me/Display All Adoptable Pets/View Controllers/AllPetDetailViewController.swift

```diff
@@ -6,6 +6,7 @@
 //
 
 import UIKit
+import Firebase
```

Comment:
The `ViewController` in this case doesn't need to know about firebase 😄

Reply:
I agree. Thank you Brandon 👍🏻